testgrid/cmd/updater/main.go

```diff
@@ -392,6 +539,37 @@ func main() {
 		}
 		close(builds)
 	}()
-	grid := ReadBuilds(builds, 1000, Days(1))
-	log.Printf("Grid: %v", grid)
+	grid := ReadBuilds(*tg, builds, 1000, Days(0.25))
+	log.Printf("Grid: %d %s", len(grid.Columns), grid.String())
+	buf, err := proto.Marshal(&grid)
+	if err != nil {
+		log.Fatalf("Failed to encode grid: %v", err)
+	}
+	var zbuf bytes.Buffer
+	zw := zlib.NewWriter(&zbuf)
+	if _, err = zw.Write(buf); err != nil {
+		log.Fatalf("Failed to compress gs://%s/%s: %v", b, o, err)
+	}
+	if err = zw.Close(); err != nil {
+		log.Fatalf("Failed to close zlib gs://%s/%s buffer: %v", b, o, err)
+	}
+	if b == "k8s-testgrid" {
+		log.Fatalf("do not change prod")
+	}
+	w := client.Bucket(b).Object(o).NewWriter(ctx)
+	w.SendCRC32C = true
+	buf = zbuf.Bytes()
+	w.ObjectAttrs.CRC32C = crc32.Checksum(buf, crc32.MakeTable(crc32.Castagnoli))
```

Comment:
Comment on why this is necessary. Also, maybe put that table in a global var block.

Reply:
Done